cron/hourly/git_maintenance_tasks_lock

```diff
@@ -4,4 +4,5 @@ set -e
 # Run git_maintenance_tasks only if not running already
 DIR="$(readlink -e "$(dirname $0)/..")"
 mkdir -p ~/lmms-repo
-flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock "$DIR/git_maintenance_tasks"
+flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock \
+	"$DIR/git_maintenance_tasks_retry"
```

Comment:
Since `pipefail` is off, can't we do this and save the redundant script?

```bash
flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock \
   "$DIR/git_maintenance_tasks" || rm -fr ~/lmms-repo; "$DIR/git_maintenance_tasks"
```